examples/clean-architecture/entities/note.entity.js

```diff
@@ -0,0 +1,43 @@
+var isEmptyString = function (str) {
+  return str === undefined || str === null || str.trim() === "";
+};
+var throwIfEmpty = function (str, field) {
+  if (isEmptyString(str)){
+    var error = new Error("field '" + field + "' should not be empty");
+    error.status = 400;
+    throw error
+  }
+};
+
+function Note(title, content, createdAt, updatedAt, id) {
```

Comment:
This could be simplified a lot with ES6 `class`, is there any good reason to not use it as its supported since v6? Same thing with `var`